framework/core/tests/integration/api/groups/ListTest.php

```diff
@@ -95,9 +95,9 @@ public function filters_only_hidden_groups_for_admin()
             $this->request('GET', '/api/groups', [
                 'authenticatedAs' => 1,
             ])
-            ->withQueryParams([
-                'filter' => ['hidden' => 1],
-            ])
+                ->withQueryParams([
+                    'filter' => ['hidden' => 1],
+                ])
```

Comment:
nitpick: just to keep the git history clean lol
```suggestion
            ->withQueryParams([
                'filter' => ['hidden' => 1],
            ])
```